Sources/MapboxDirections/TollPrice.swift

```diff
@@ -0,0 +1,136 @@
+import Foundation
+import Turf
+
+/**
+ :nodoc:
+ Information about toll payment method.
+ */
+public struct PaymentMethod: Hashable, Equatable {
+    /**
+     Method identifier.
+     */
+    public let identifier: String
+    
+    /**
+     Payment is done by electronic toll collection.
+     */
+    public static let electronicTollCollection = PaymentMethod(identifier: "etc")
+    /**
+     Payment is done by cash.
+     */
+    public static let cash = PaymentMethod(identifier: "cash")
+}
+
+/**
+ :nodoc:
+ Categories by which toll fees are divided.
+ */
+public struct Category: Hashable, Equatable {
```

Comment:
This is a very generic name prone to collisions with unrelated types in other libraries. Let’s call it `TollCategory`.